tdrs-frontend/src/components/UploadReport/UploadReport.jsx

```diff
@@ -2,6 +2,10 @@ import React, { useEffect, useRef, useState } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
 import PropTypes from 'prop-types'
 import { fileInput } from 'uswds/src/js/components'
+import {
+  getAvailableFileList,
+  triggerDownloadDialog,
```

Comment:
This import is unused.

Reply:
![Screenshot from 2021-06-02 19-05-49](https://user-images.githubusercontent.com/22626085/120563000-93cbf600-c3d5-11eb-8541-f90c10399b2a.png)
I would remove this block and then add `getAvailableFileList` to the `clearError` import below